### PR TITLE
pull yelp sample data and fix package-lock.json issue v3

### PR DESCRIPTION
**Pull request by tedgreen29**

### CODE REVIEW

**Comment by tedgreen29** [comment]
This cannot be rebased, but can be merged into master. I've been trying to rebase but no dice, so merge is only possible/necessary. Sorry if I messed something up.

File: tm/tm.config.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const key = 'dijuQaQgNxRJLxYmi5ZlROnW7RG0c4kC';
```
Comment (dmukherjee) [inline tm/tm.config.js]: @tedgreen29 This is a public repo. I don't think you want to check-in your key 😃

Comment (tedgreen29) [reply tm/tm.config.js]: I could have gitignored the file, but I wanted you guys to be able to use this for now. By the end of the project, I plan to remove this file and provide instructions on how to fill it.

Comment (dmukherjee) [reply tm/tm.config.js]: Thats ok if you want it to be in there. Just wanted to make sure!

**Pull request merged**